ijPlugin/src/main/kotlin/org/jetbrains/research/tasktracker/requests/FileRequests.kt

```diff
@@ -0,0 +1,47 @@
+package org.jetbrains.research.tasktracker.requests
+
+import com.intellij.openapi.diagnostic.Logger
+import io.ktor.client.*
+import io.ktor.client.engine.cio.*
+import io.ktor.client.request.forms.*
+import io.ktor.http.*
+import kotlinx.coroutines.runBlocking
+import org.jetbrains.research.tasktracker.tracking.Loggable
+import org.jetbrains.research.tasktracker.ui.main.panel.storage.MainPanelStorage
+import java.io.File
+
+object FileRequests {
+
+    private val client = HttpClient(CIO)
+    private val logger: Logger = Logger.getInstance(FileRequests::class.java)
+
+    fun Loggable.send() = this.getLogFiles().all {
+        sendFile(it, this.subDir)
+    }
+
+    @Suppress("TooGenericExceptionCaught")
+    private fun sendFile(file: File, subdir: String) = runBlocking {
```

Comment:
I'd suggest to setup a size limit for files

Reply:
Also, I'd suggest adding authentication and accepting files only if the auth token was sent.